src/components/LinkItems/ItemIcon.vue

```diff
@@ -25,6 +26,8 @@ import { faviconApi as defaultFaviconApi, faviconApiEndpoints, iconCdns } from '
 import EmojiUnicodeRegex from '@/utils/EmojiUnicodeRegex';
 import emojiLookup from '@/utils/emojis.json';
 
+const simpleIcons = require('simple-icons');
```

Comment:
This should be: `import simpleIcons from 'simple-icons';` 
In order to allow for webpack tree shaking

Reply:
Done